main.cpp

```diff
@@ -1,4 +1,5 @@
 #include <iostream>
+using namespace std;
```

Comment:
You should never use namespace. 
Literarly 5 seconds in google:
https://stackoverflow.com/questions/1452721/why-is-using-namespace-std-considered-bad-practice/1452738#1452738